python/lsst/ip/isr/isrMockLSST.py

```diff
@@ -724,6 +843,26 @@ def makeBfKernel(self):
 
         return bfKernelObject
 
+    def makeDeferredChargeCalib(self):
+        """Generate a CTI calibration.
+
+        Returns
+        -------
+        cti : `lsst.ip.isr.deferredCharge.DeferredChargeCalib`
+            Simulated deferred charge calibration.
+        """
+
+        metadataDict = {'metadata': PropertyList()}
+        metadataDict['metadata'].add(name="OBSTYPE", value="CTI")
+        metadataDict['metadata'].add(name="CALIBCLS",
+                                     value="lsst.ip.isr.deferredCharge.DeferredChargeCalib")
+        metadataDict['metadata'].add("USEGAINS", value=False)
```

Comment:
Can you add a comment here that this should always be False for the new ISR task because it's all done in electrons?